### PR TITLE
DOCS Improvements: Update Old Screenshots, Add Codeblocks.

### PR DESCRIPTION
**Pull request by crydotsnake**

### CODE REVIEW

File: docs/07. media library/03. integrating in your module.md [hunk 1]
```diff
@@ -115,7 +116,7 @@ Just pass the media group entity to your custom entity constructor.
 
 ### In your Update Command class
 
-```php
+``php
```
Comment (carakas) [inline docs/07. media library/03. integrating in your module.md:119]: you accidentally removed a backtick here

**Review by carakas — CHANGES_REQUESTED** [review]
Thank you so much for cleaning up some of our docs.
I have a couple of feedback points.

- In docs/03. user guide/assets/editor_embed.png you entered the iframe code instead of the URL of the iframe in the URL field, does that work? it looks a bit weird in the screenshot.
- Your screenshot of youtube  docs/03. user guide/assets/editor_youtubecode.png  is in german. Could you replace this with an English one since the docs are in English?
-

File: docs/07. media library/03. integrating in your module.md (hunk 1 above)
Comment (crydotsnake) [reply docs/07. media library/03. integrating in your module.md]: Fixed it

**Comment by crydotsnake** [comment]
> Thank you so much for cleaning up some of our docs.
> I have a couple of feedback points.
> 
> * In docs/03. user guide/assets/editor_embed.png you entered the iframe code instead of the URL of the iframe in the URL field, does that work? it looks a bit weird in the screenshot.
> * Your screenshot of youtube  docs/03. user guide/assets/editor_youtubecode.png  is in german. Could you replace this with an English one since the docs are in English?

I updated the editor_embed.png image

I changed the screenshot `editor_youtubecode.png` too English

**Pull request merged**